README.md

```diff
@@ -84,7 +84,7 @@ Due to an issue with ESLint, config extension cannot be called from a globally i
 - [ESLint](https://github.com/eslint/eslint) 2.10.2
 - [eslint-plugin-react](https://www.npmjs.com/package/eslint-plugin-react) 6.0.0
 - [eslint-plugin-filenames](https://www.npmjs.com/package/eslint-plugin-filenames) 1.1.0
-- [eslint-plugin-import](https://www.npmjs.com/package/eslint-plugin-import) 1.12.0
+- [eslint-plugin-import](https://www.npmjs.com/package/eslint-plugin-import) 1.13.0
```

Comment:
Nothing was changed between 12 -> 13 besides fixes, the config we used in https://github.com/FormidableLabs/eslint-config-formidable/pull/18 is `1.13.0` and this line should have been updated there but I hadn't realized they released a version since working on it yesterday. This is intended to update the readme to reflect the correct config version.